acceptance/tests/direct_puppet/cached_catalog_remediate_local_drift.rb

```diff
@@ -89,16 +89,18 @@
       assert_equal(r, '0', "reports directory should be empty!")
 
       step "Verify puppet run without drift does not make file request from server"
-      r = on(agent, puppet("agent",
+      on(agent, puppet("agent",
         "--use_cached_catalog",
         "--server", "no_such_host",
         "--report_server", master.hostname,
         "--onetime",
         "--no-daemonize",
         "--detailed-exitcodes",
         "--verbose"
-      )).stderr
-      assert_equal(r, "", "Fail: Did agent try to contact server?")
```

Comment:
That particular step of the test does not issue any deprecation warnings on Windows:
```
* Verify puppet run without drift does not make file request from server
  
  qkdrnn9879m6bnl.delivery.puppetlabs.net (windows2012r2-64-1) 18:41:43$ cmd.exe /c puppet agent --use_cached_catalog --server no_such_host --report_server x6rgarwz0fz5piz.delivery.puppetlabs.net --onetime --no-daemonize --detailed-exitcodes --verbose
    Info: Using cached catalog from environment 'production'
    Info: Applying configuration version '1490294499'
    Notice: Applied catalog in 0.05 seconds
  
  qkdrnn9879m6bnl.delivery.puppetlabs.net (windows2012r2-64-1) executed in 9.28 seconds
  Warning: 
  the audit paramter should issue a deprecation warning.
  Expected /The `audit` metaparameter is deprecated/ to match # encoding: US-ASCII
  "".
```

I'm going to update this to just assert that there are no errors.